composer.json

```diff
@@ -6,9 +6,11 @@
     "test": "vendor/bin/phpunit"
   },
   "require": {
-    "php": "^7.0 || ^8.0",
-    "amphp/http-server": "^1.0 || ^2.0"
+    "php": ">= 8.1",
+    "amphp/http-server": "^3"
   },
+  "minimum-stability": "dev",
```

Comment:
Mimic amphp strategy: https://github.com/amphp/http-server-router/blob/v2/composer.json#L44 to be able to use directly v3

Reply:
I would prefer we set this to `beta`. I only want to install released tags.

Reply:
Ok. I'll merge this in as `dev`. I'm not gonna cut a release until we get a beta tag that we can install. That means to use this with newest amphp/http-server you'll need to install `dev-main`. Since a project using the newest amphp/http-server is probably already doing this I don't think this is a large burden on the end user.